linux/user-and-file-management/linux-intro/hello-linux.md

```diff
@@ -0,0 +1,51 @@
+---
+author: kapnobatai136
+
+aspects:
+  - introduction
+
+type: normal
+
+category: must-know
+
+tags:
+  - introduction
+  - linux
+
+links:
+  - '[Install Ubuntu](https://ubuntu.com/tutorials/tutorial-install-ubuntu-desktop#1-overview){website}'
+  - '[Try Ubuntu](https://ubuntu.com/tutorials/try-ubuntu-before-you-install#1-getting-started){website}'
+
+---
+
+# Hello Linux
+
+---
+## Content
+
+All this talk, and we still haven't shown you how to use Linux 😅.
+
+> Note: we're going to assume you have an installed Linux distribution. If you don't, check out the *Learn More* section for Ubuntu installation guides.
```

Comment:
Sometimes you use Note and sometimes **Note** in these callouts.

We should be consistent with the font-weight but why not use an emoji instead?

One idea:

```suggestion
> 💡We're going to assume you have an installed Linux distribution. If you don't, check out the *Learn More* section for Ubuntu installation guides.
```